presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Release 335 (XX May 2020)
```

Comment:
12 June or whatever .. which makes the line longer too..

Reply:
We've been using the 3-letter abbreviations for these